### ISSUE TITLE
[CapOne] Request for patch to unblock CapOne

### ISSUE THREAD

**Issue by ryphil** [sourcegraph/sourcegraph#22358]
@sourcegraph/distribution I am requesting the following commits be included in a patch release. They are already merged into `main`:

The intent of the questions below is to ensure we keep Sourcegraph high quality and [only create patch releases based on a strict criteria.](https://about.sourcegraph.com/handbook/engineering/releases#when-are-patch-releases-performed) If you can answer yes to many or most of these questions, we will be happy to create the patch release.

- https://github.com/sourcegraph/customer/issues/335

I have read [when and why we perform patch releases](https://about.sourcegraph.com/handbook/engineering/releases#when-are-patch-releases-performed) and answer the questions as follows:

> Are users/customers actively asking us for these changes and cannot wait until the next full release?

Yes, this is a blocking issue for Capital One. Unblock plan linked here: https://docs.google.com/spreadsheets/d/1pWbDH6yIWWzyIVQIu8hqTOByjzudLQYGLDCqCTjlEvU/edit#gid=0 

> Are the changes extremely minimal, well-tested, and low risk such that not testing as we do in a full release is OK?

Yes. More details can be found in the GitHub issue: https://github.com/sourcegraph/customer/issues/335

> Is there some functionality completely broken that warrants redacting the prior release of Sourcegraph and advising users wait for the patch release?

No. This this is providing a fix for a bug in GitHub's code. We have notified GitHub but lack of movement required our team to provide a fix to unblock Capital One. 

> This will interrupt our regular planned work and release cycle, taking one full working day of our time, and will take up all of our site admin's valuable time by asking them to upgrade or producing noise for them if they don't need to upgrade.
>
> Do you believe the changes are important enough to warrant this?

Yes. Capital One has been waiting for ~3-4 weeks for a fix from GitHub and to date we do not see movement on this issue. The work associated with this patch is provide a workaround for the GitHub issue, ultimately unblocking Capital One to move forward with Sourcegraph POC. 

> Patch releases are a signal we can do something better to improve the quality of Sourcegraph. Have you already scheduled a call (or created a google doc) to perform a [retrospective](https://about.sourcegraph.com/retrospectives) and identify ways we can improve?

No. This issue is necessary due to a bug in GitHub's platform. Due to the lack of movement in the GitHub issue, we decided with CapOne to move forward with Plan B from the Sourcegraph side. This 

---

**For the [release captain](https://about.sourcegraph.com/handbook/engineering/releases#release-captain)** - after reviewing and approving this request:

- If there is [already an upcoming patch release](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Arelease-tracking+), add the listed commits alongside a link to this issue
- If there is no upcoming patch release, create a new one:
  - Update [`dev/release/release-config.jsonc`](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/dev/release/release-config.jsonc) with the patch release in `upcomingRelease` and `releaseDate` (and open a PR to `main` to update it)
  - `yarn release tracking:issues`
  - Add the listed commits alongside a link to this issue to the generated [release tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Arelease-tracking+)

Comment and close this issue once the relevant commit(s) have been cherry-picked into the release branch.
